<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Series Satellite Data Visualizer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Data/globe.ico</normaloff>Data/globe.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>View Mode</string>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Single </string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dual</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>No Data value</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Boundary File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="toolTip">
     <string>Enter Shapefile Path</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Optional</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set Min-Max Value</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Set Min-Max value</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fix legend</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Auto Detect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter Folder Fath</string>
    </property>
    <property name="text">
     <string>Data/clip</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Controls</string>
     </property>
     <property name="text">
      <string>Controls</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Automatic</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Manual</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select Stretching</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stretching </string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Select Stretching</string>
     </property>
     <item>
      <property name="text">
       <string>Min-Max</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Histogram Equlization</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 Inter Quartile Range</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Standard Deviation </string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Percentile Clipping</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Enter the Values</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>1 or 1,2</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Browse Folder Path</string>
    </property>
    <property name="text">
     <string> ...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Browse Shape File</string>
    </property>
    <property name="text">
     <string> ...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Show</string>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>CLose</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Click for First File</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Click for Previous</string>
     </property>
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Click for Next</string>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Reset</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Selet Colormap</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Colormap</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Selet Colormap</string>
     </property>
     <item>
      <property name="text">
       <string>Blues</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BuGn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BuPu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GnBu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Greens</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Greys</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Oranges</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OrRd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PuBu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PuBuGn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PuRd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Purples</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RdPu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Reds</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>YlGn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>YlGnBu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>YlOrBr</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>YlOrRd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>afmhot</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>autumn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>bone</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cool</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>copper</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gist_heat</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gray</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>hot</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>pink</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spring</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>summer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>winter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BrBG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>bwr</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>coolwarm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PiYG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PRGn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PuOr</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RdBu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RdGy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RdYlGn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Spectral</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>seismic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Accent</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dark2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Paired</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pastel1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pastel2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Set1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Set2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Set3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gist_earth</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>terrain</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ocean</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gist_stern</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>brg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CMRmap</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cubehelix</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gnuplot</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gnuplot2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gist_ncar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nipy_spectral</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>jet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>rainbow</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gist_rainbow</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>hsv</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>flag</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>prism</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Extent</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
